protos/profiles.proto

```diff
@@ -19,10 +19,16 @@ message Profile {
 
 message JDBC {
   string host = 2;
-  int64 port = 3;
```

Comment:
My understanding was these are backwards compatible, so you could just change this and save quite a bit of work here? https://developers.google.com/protocol-buffers/docs/proto#updating

No one has set a non 32-bit port number so we don't need to worry about that :)

Reply:
Interesting, I didn't know this was safely change-compatible, good to know.